packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -603,6 +604,13 @@ class TextureLayer extends Layer {
   /// un-freezes it when it is certain that a frame with the new size is ready.
   final bool freeze;
 
+  /// Used to set the filterQuality of the texture.
+  ///
+  /// Use the [FilterQuality.low] quality setting to scale the texture, which corresponds to
+  /// bilinear interpolation, rather than the default [FilterQuality.none] which corresponds
```

Comment:
I think this PR is going to change the default quality to `FilterQuality.low` instead of `FilterQuality.none`? Also try the Dart doc template to avoid duplicates: https://github.com/dart-lang/dartdoc#macros